client-app/shared/cart/components/cart-item-for-later.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <VcProductCard :background="false">
+    <VcProductImage :img-src="item.imageUrl" :alt="item.product!.name" />
+
+    <VcProductTitle lines-number="2" fix-height :to="link" :title="item.product!.name" @click="$emit('linkClick', $event)">
```

Comment:
try to avoid item.product!.name -> item.product.name
add some checks on the top